### PR TITLE
remove test dependency on CSV

### PR DESCRIPTION
**Pull request by bjarthur**
- [x] I've run the regression tests

only one test used it, so i refactored to use DelimitedFiles in the standard library instead.

### CODE REVIEW

File: test/testscripts/issue120.jl [hunk 1]
```diff
@@ -1,5 +1,7 @@
-using Gadfly, DataFrames, Compat, CSV
+using Gadfly, DataFrames, Compat, DelimitedFiles
```
Comment (Mattriks) [inline test/testscripts/issue120.jl]: Why is there a `Compat ` here?

File: test/testscripts/issue120.jl [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
 set_default_plot_size(6inch, 3inch)
 
-plot(CSV.read(joinpath(dirname(@__DIR__), "data","issue120.csv")), x=:x1, y=:x2)
+dlm = readdlm(joinpath(dirname(@__DIR__), "data","issue120.csv"), ',')
```
Comment (Mattriks) [inline test/testscripts/issue120.jl]: or in this style (using `header=true`)
```julia
dlm, varnames =  readdlm( joinpath(...), ',', header=true)
```
which will simplify the next line (no convert needed for `dlm`)

**Comment by Mattriks** [comment]
I just looked at #120, and it seems that the point was to test the syntax:
```julia
plot(readtable("test.csv"), x="x1", y="x2")
```
`readtable` produced a dataframe, so replacing with `DelimitedFiles.readdlm` (and constructing the dataframe outside the plot call) here isn't really the same.

File: test/testscripts/issue120.jl (hunk 2 above)
Comment (Mattriks) [reply test/testscripts/issue120.jl]: Ignore my deleted remark on #120, I realise now #120 was a test about precision.

**Comment by bjarthur** [comment]
i'm working on a second PR which removes all Compat.  no longer necessary now that we don't support 0.6.

**Pull request merged**